### DISCUSSION TITLE
The mobile app don’t support mermaid and theme context for images

### DISCUSSION THREAD

**Discussion by bric3** [community/community#27434] [Mobile]
The issues, pull requests, read-mes don't support all features of the web platform.

I noticed in particular 
* that images that switch depending on color scheme context, see https://github.com/community/community/discussions/16910
* mermaid diagrams are not rendered

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.74.0 (74357630)
Device: iPhone 13 (iOS 16.0)
</details>

**Comment by aconsuegra** [comment]
Thanks for reaching out and your feedback @bric3! I submitted the feedback request to support Mermaid diagrams to our Product team so we can prioritize it. Regarding the support of images that adapt to the selected theme, that should be already supported for issues and pull requests, are you experiencing issues only with discussions?

**Reply by bric3** [reply]
In rendered READMEs the `picture` element don't show the image.
In discussions this seem OK, for example I can see the images in the linked discussion.

**Reply by aconsuegra** [reply]
Understood! Thanks for following up

**Reply by jhildenbiddle** [reply]
Hi @aconsuegra. Any updates on relative URLs used with the `<picture>` on the iOS app? I now have a handful of repos with README files that leverage the `<picture>` element and all of them render broken images on iOS. For example, the [Typora Themeable](https://github.com/jhildenbiddle/typora-themeable) repo's README contains a screenshot that renders as expected when viewed in a browser but displays a broken image placeholder in the iOS app.

Thanks!

**Reply by aconsuegra** [reply]
No, sorry that we don't have updates here yet

**Comment by johnbent** [comment]
Hello, I would also like mermaid images to render in the GitHub app please. I'm trying to convince a team to switch to use markdown for documentation and this is an important feature.
